api/src/functionalTest/java/uk/gov/hmcts/payment/functional/fixture/PaymentFixture.java

```diff
@@ -58,6 +58,75 @@ public static CreditAccountPaymentRequest aPbaPaymentRequest(String amountString
             .build();
     }
 
+    public static CreditAccountPaymentRequest aPbaPaymentRequestForUnspec(String amountString, Service service) {
+        String ccdCaseNumber = "1111-CC12-" + RandomUtils.nextInt();
+        return CreditAccountPaymentRequest.createCreditAccountPaymentRequestDtoWith()
+            .amount(new BigDecimal(amountString))
+            .description("New passport application")
+            .ccdCaseNumber(ccdCaseNumber)
+            .caseReference("aCaseReference")
+            .service(service)
+            .currency(CurrencyCode.GBP)
+            .siteId("AAA7")
+            .customerReference("CUST101")
+            .organisationName("ORG101")
+            .accountNumber("PBA0082848")
+            .fees(Lists.newArrayList(
+                FeeDto.feeDtoWith()
+                    .calculatedAmount(new BigDecimal(amountString))
+                    .code("FEE0001")
+                    .version("1")
+                    .build())
+            )
+            .build();
+    }
+
+    public static CreditAccountPaymentRequest aPbaPaymentRequestForIAC(String amountString, Service service) {
```

Comment:
Can we combine all these mock request into one dynamic by passing Service Specific SiteIds, we can reduce a good amount of duplicate codes

Reply:
Points noted. We will look into this in the next release.